Description: Avoid build dependency on textlive-fonts-extra
 The tex sources seem to require this universe package to use
 DejaVuSansMono fonts. Replacing this with simple courier so
 we can drop the build-dependency.
 The resulting pdf documents seem to be 
Forwarded: No
Author: Stefan Bader <stefan.bader@canonical.com>
Last-Update: 2015-11-17

Index: dpdk/doc/guides/conf.py
===================================================================
--- dpdk.orig/doc/guides/conf.py	2015-11-17 11:16:46.427330439 +0100
+++ dpdk/doc/guides/conf.py	2015-11-17 14:35:46.203281665 +0100
@@ -52,7 +52,7 @@ latex_documents = [
 # Latex directives to be included directly in the latex/pdf docs.
 latex_preamble = r"""
 \usepackage[utf8]{inputenc}
-\usepackage{DejaVuSansMono}
+\usepackage{courier}
 \usepackage[T1]{fontenc}
 \usepackage{helvet}
 \renewcommand{\familydefault}{\sfdefault}
